### PR TITLE
Miscellaneous edits throughout the mansucript

### PR DESCRIPTION
**Pull request by dhimmel**

### CODE REVIEW

File: content/02.main-text.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ We'll discuss our experience with our open review including the pros and cons of
 We also discuss the extent to which such efforts can contribute to a living and frequently updated literature.
 `TODO: discuss stage 2 maintainers`
 
-We initiated our open online review article by creating a GitHub repository (https://github.com/greenelab/deep-review) to coordinate and manage contributions.
+We initiated our open online review article by creating a GitHub repository (<https://github.com/greenelab/deep-review>) to coordinate and manage contributions.
```
Comment (cgreene) [inline content/02.main-text.md]: What do angle brackets do? Make a hyperlink?

File: content/02.main-text.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Writing review articles in a public forum allows review authors to engage with t
 In addition, discussing manuscripts in the open provides one form of pre- and post-publication peer review [@doi:10.7554/eLife.38532], incentivizing the reviews with potential manuscript authorship.
 However, inviting wide authorship brings many technical and social challenges such as how to fairly distribute credit, coordinate the scientific content, and collaboratively manage extensive reference lists.
 
-To coordinate this effort, we developed a manuscript writing process using the Markdown language, [GitHub](https://github.com/greenelab/deep-review/), and our new [Manubot](https://github.com/greenelab/manubot-rootstock) tool for automating manuscript generation.
+To coordinate this effort, we developed a manuscript writing process using the Markdown language, [GitHub](https://github.com/greenelab/deep-review), and our new [Manubot](https://github.com/greenelab/manubot-rootstock) tool for automating manuscript generation.
```
Comment (cgreene) [inline content/02.main-text.md]: Is it odd that the GitHub link goes specifically to our review?

File: content/02.main-text.md [hunk 3]
```diff
@@ -96,10 +96,11 @@ When changes occur, the CI service generates an updated manuscript.
...
 Since the HTML manuscript is hosted using [GitHub Pages](https://pages.github.com/), the CI service automatically deploys the new manuscript version when it pushes the updated outputs to GitHub.
 
-For this article, the source GitHub repository is https://github.com/greenelab/meta-review.
+For this article, the source GitHub repository is <https://github.com/greenelab/meta-review>.
 When this repository changes, Travis CI [rebuilds](https://travis-ci.org/greenelab/meta-review) the manuscript.
 If successful, the output is deployed back to GitHub (to dedicated [`output`](https://github.com/greenelab/meta-review/tree/output) and [`gh-pages`](https://github.com/greenelab/meta-review/tree/gh-pages) branches).
-As a result, https://greenelab.github.io/meta-review stays up to date with the latest HTML manuscript.
+As a result, <https://greenelab.github.io/meta-review> stays up to date with the latest HTML manuscript.
+Furthermore, versioned URLs, such as <https://greenelab.github.io/meta-review/v/3ec66cd440e78dac33f4700f1cecbc4d95b20139/>, preserve access to previous manuscript versions.
```
Comment (agitter) [inline content/02.main-text.md]: Do we want a `TODO` to update this to a more polished version?

Comment (cgreene) [inline content/02.main-text.md]: `preserve` -> `provide`?

**Review by cgreene — APPROVED** [review]
A couple questions. No objections that rise to the level of requiring changes though.

File: content/02.main-text.md (hunk 2 above)
Comment (agitter) [reply content/02.main-text.md]: I removed that link in #54

File: content/02.main-text.md (hunk 1 above)
Comment (dhimmel) [reply content/02.main-text.md]: Without them, these URLs are printed as text without any hyperlink. This makes the URLs hyperlink to themselves

**Review by agitter — APPROVED** [review]
Looks good to me.

**Pull request merged**